pkg/statefulset/statefulset.go

```diff
@@ -205,6 +205,30 @@ func (builder *Builder) Exists() bool {
 	return err == nil || !k8serrors.IsNotFound(err)
 }
 
+// Delete a statefulset from the cluster.
+func (builder *Builder) Delete() error {
+	if valid, err := builder.validate(); !valid {
+		return err
+	}
+
+	glog.V(100).Infof("Deleting statefulset %s in namespace %s", builder.Definition.Name, builder.Definition.Namespace)
+
+	if !builder.Exists() {
+		return nil
```

Comment:
Add glog message

Reply:
Other Delete() methods don't use a glog for a resource not existing:
- [deployment](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/deployment/deployment.go#L430-L434)
- [daemonset](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/daemonset/daemonset.go#L293-L297)
- [configmap](https://github.com/openshift-kni/eco-goinfra/blob/main/pkg/configmap/configmap.go#L127-L129)

Reply:
That has to be improved. Please refer to: https://github.com/openshift-kni/eco-goinfra/blob/411277cec65bd7a730c2bdf9938ad61557eca100/pkg/nmstate/nmstate.go#L131

Reply:
Done!